Buttplug.Server.Managers.WinUSBManager/RezTranceVibratorDevice.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Buttplug.Core;
+using Buttplug.Core.Messages;
+using MadWizard.WinUSBNet;
+
+namespace Buttplug.Server.Managers.WinUSBManager
+{
+    class RezTranceVibratorDevice : ButtplugDevice
+    {
+        private USBDevice _device;
+        private static uint _vibratorCount = 1;
+
+        public RezTranceVibratorDevice(IButtplugLogManager aLogManager, USBDevice aDevice, uint aIndex)
+            : base(aLogManager, "Trancevibrator " + aIndex, "Trancevibrator " + aIndex)
+        {
+            _device = aDevice;
+            MsgFuncs.Add(typeof(SingleMotorVibrateCmd), new ButtplugDeviceWrapper(HandleSingleMotorVibrateCmd));
+            MsgFuncs.Add(typeof(StopDeviceCmd), new ButtplugDeviceWrapper(HandleStopDeviceCmd));
+            MsgFuncs.Add(typeof(VibrateCmd), new ButtplugDeviceWrapper(HandleVibrateCmd));
+        }
+
+        private Task<ButtplugMessage> HandleStopDeviceCmd(ButtplugDeviceMessage aMsg)
+        {
+            BpLogger.Debug("Stopping Device " + Name);
+            return HandleSingleMotorVibrateCmd(new SingleMotorVibrateCmd(aMsg.DeviceIndex, 0, aMsg.Id));
+        }
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        private async Task<ButtplugMessage> HandleVibrateCmd(ButtplugDeviceMessage aMsg)

```

Comment:
If you return the responses like `return Task.FromResult<ButtplugMessage>(new Ok(aMsg.Id));` you can remove the async modifier.